ui/src/views/OpenVPNRW.vue

```diff
@@ -658,64 +658,88 @@
                 <span class="field-section-toggle-pf">{{$t('openvpn_rw.extra_params')}}</span>
               </legend>
 
-              <div
+              <div 
                 v-show="newConfiguration.advanced"
+                :class="['form-group',newConfiguration.errors.PushDhcpOptionsStatus.hasError ? 'has-error' : '']"
+                >
+                <label
+                  class="col-sm-3 control-label"
+                  for="textInput-modal-markup"
+                >{{$t('openvpn_rw.PushDhcpOptionsStatus')}}</label>
+                <div class="col-sm-9">
+                  <toggle-button
+                    class="min-toggle"
+                    :width="40"
+                    :height="20"
+                    :color="{checked: '#0088ce', unchecked: '#bbbbbb'}"
+                    :value="newConfiguration.PushDhcpOptionsStatus"
+                    :sync="true"
+                    @change="togglePushDhcpOptionsStatus()"
+                  />
+                  <span
+                    v-if="newConfiguration.errors.PushDhcpOptionsStatus.hasError"
+                    class="help-block"
+                  >{{$t('validation.validation_failed')}}: {{$t('validation.'+newConfiguration.errors.PushDhcpOptionsStatus.message)}}</span>
+                </div>
+              </div>
+              <div
+                v-show="newConfiguration.advanced && newConfiguration.PushDhcpOptionsStatus"
                 :class="['form-group', newConfiguration.errors.PushDomain.hasError ? 'has-error' : '']"
               >
                 <label
                   class="col-sm-3 control-label"
```

Comment:
To have correctly aligned items on the form we have to move `class="col-sm-3 control-label"` to `class="col-sm-5 control-label"`